DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -347,6 +347,20 @@ def run(self):
     # configure geometry creation
     self.geometry.constructGeometry(kernel, geant4, self.output.geometry)
 
+    # ----------------------------------------------------------------------------------
+    # Configure run, event, track, step, and stack actions, if present
+    for action_list, DDG4_Action, kernel_Action in \
+        [(self.action.run, DDG4.RunAction, kernel.runAction),
+         (self.action.event, DDG4.EventAction, kernel.eventAction),
+         (self.action.track, DDG4.TrackingAction, kernel.trackingAction),
+         (self.action.step, DDG4.SteppingAction, kernel.steppingAction),
+         (self.action.stack, DDG4.StackingAction, kernel.stackingAction)]:
+      for action_dict in action_list:
+        action = DDG4_Action(kernel, action_dict["name"])
+        for parameter, value in action_dict['parameter'].items():
+          setattr(action, parameter, value)
+        kernel_Action().add(action)
+
```

Comment:
We may want to `kernel.registerGlobalAction(action)` here to allow access by name later on(?).